### ISSUE TITLE
Is aio-libs.discourse.group dead?

### ISSUE THREAD

**Issue by Dreamsorcerer** [fafhrd91/aiohttp#7141]
### Discussed in https://github.com/aio-libs/aiohttp/discussions/7140 (@pepoluan)

<div type='discussions-op-text'>

<sup>Originally posted by **pepoluan** December 23, 2022</sup>
Though I still find references to it if I do a search, trying to access https://aio-libs.discourse.group/ fails, and using down detectors all seem to agree that it's down.

Is it going to be a permanent down or just a temporary one?
</div>

**Comment by Dreamsorcerer** [comment]
I assume @asvetlov set this up. We should obviously remove it from the docs if it is no longer intended to be running.

**Comment by webknjaz** [comment]
I've found a private discussion dating Oct 30-31, 2021 on Gitter where we agreed that GitHub Discussions is a way to go, and it doesn't make sense to keep the community fragmented. We talked about sunsetting Discourse and Google Groups, but it seems there was no action taken after that, short of putting https://github.com/aio-libs/.github/discussions as an org URL. Looks like I also edited https://github.com/aio-libs/aiohttp/issues/4257 on Nov1, 2021, replacing the link there. Dunno why/when Discourse stopped being available. I think I've lost my privileges there at some point, randomly. But I stopped checking it out long ago.

So the action item would be to start cleaning up the links, removing references to the said locations.
